modules/labs-react/common/stories/Box.stories.mdx

```diff
@@ -0,0 +1,92 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Box} from '@workday/canvas-kit-labs-react/common';
+// examples
+import {Border} from './examples/Border';
+import {Color} from './examples/Color';
+import {Depth} from './examples/Depth';
+import {Layout} from './examples/Layout';
+import {Position} from './examples/Position';
+import {Space} from './examples/Space';
+
+<Meta title="Labs/Common/Box" component={Box} />
+
+# Box
+
+[![LABS: Beta](https://img.shields.io/badge/LABS-beta-orange)](https://github.com/Workday/canvas-kit/tree/master/modules/labs-react/README.md)
+This component is work in progress and currently in pre-release.
+
+`Box` is a primitive component that provides a common, ergonomic API around Canvas design tokens. It
+is highly flexible, and its primary purpose is to build other components.
+
+## Style Props
+
+While `Box` supports `as`, `ref`, and all semantic element props, style props are at its heart.
+Style props are special design-system-aware props that allow you to use Canvas design tokens
+efficiently as you build components. Style props are better understood by their categories, which
+are listed below. Each category has a more detailed explanation in the following subsections.
+
+### Categories
+
+- [Border](#border)
+- [Color](#color)
+- [Depth](#depth)
+- [Layout](#layout)
+- [Position](#position)
+- [Space](#space)
+
+### Border
+
+Border style props allow you to adjust the border styles of components. Some props, such as
+`borderRadius` and `borderColor`, are connected to our design tokens. This makes it really simple to
+use Canvas values as you build UI.
```

Comment:
I expected to see a listing of border props here, especially after you enumerated the color props below ("There are three color props available: ..."), but looking at the length of the props table at the bottom I understand why you didn't.

Reply:
Yep. I agree. Ideally someone could look at any section without having to refer to the (long) props table. I'll try to think about a better way to format it.